pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "tf-playwright-stealth"
+packages = [
+    { include = "playwright_stealth" },
+]
+version = "1.0.0"
+description = "Makes playwright stealthy like a ninja!"
+authors = []
+homepage = "https://www.agentql.com/"
+repository = "https://github.com/tinyfish-io/tf-playwright-stealth"
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+playwright = "^1"
+agentql = "^1.3.0"
```

Comment:
why does it depend on agentql? It shouldn't depend on agentql

Reply:
My bad, this should be a dev dependency for tests